recipes/yojimbo/all/conanfile.py

```diff
@@ -1,117 +1,76 @@
 import os
-from conans import ConanFile, MSBuild, AutoToolsBuildEnvironment, tools
-from conans.errors import ConanInvalidConfiguration
-import yaml
+from conan import ConanFile
+from conan.tools.files import copy, get, replace_in_file
+from conan.tools.layout import basic_layout
+from conan.tools.premake import Premake, PremakeDeps, PremakeToolchain
+
+
+required_conan_version = ">=2.18.0"
```

Comment:
```suggestion
required_conan_version = ">=2.19.0"
```
Looks like https://github.com/conan-io/conan/pull/18572 is required to fix this build issue.

`Error: C:/workspace/cci_prod_PR-27827/conan-home/p/b/yojim66f270c3c143d/b/build-release/conanfile.premake5.lua:3: invalid escape sequence near '"C:\w'`

Reply:
Oh yes thank you, 

I'm aware of it @Werni2A!
I was just trying to fix Linux CI.